tests/baselines/reference/systemModuleEnum.js

```diff
@@ -0,0 +1,25 @@
+//// [systemModuleEnum.ts]
+
+export enum Foo {}
+
+export module Bar {
+	export var baz = (): any => {}
+}
+
+
+//// [systemModuleEnum.js]
+System.register([], function(exports_1) {
+    var Foo, Bar;
+    return {
+        setters:[],
+        execute: function() {
+            (function (Foo) {
+            })(Foo || (Foo = {}));
+            exports_1("Foo", Foo);

```

Comment:
You're right JS treats it as call statement. I think you can just add a semicolon to this line and you can submit a fix :+1: 
```typescript
if (compilerOptions.module === ModuleKind.System && (node.flags & NodeFlags.Export)) {
   // write the call to exporter for enum
   writeLine();
   write(`${exportFunctionForFile}("`);
   emitDeclarationName(node);
   write(`", `);
   emitDeclarationName(node);
   write(")"); // here
}
```
https://github.com/Microsoft/TypeScript/blob/master/src/compiler/emitter.ts#L4506